### PR TITLE
TELCODOCS-388: Bug Fix additions to Release Notes

### PR DESCRIPTION
**Pull request by tmulquee**
https://issues.redhat.com/browse/TELCODOCS-388

For 4.10 only. 

Preview:  https://deploy-preview-40690--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes

### CODE REVIEW

File: modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc [hunk 1]
```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
```
Comment (ctauchen) [inline modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc:1]: @tmulquee This file to be feature content. If it's feature content, it needs to go in a separate PR that you open against `main`.
CC @stevsmit

Comment (stevsmit) [reply modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc:1]: I agree. Seems like it's more suitable for a separate PR given the PR title.

Comment (tmulquee) [reply modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc:1]: Thanks, it's a feature for which I'm doing the RN and docs. I will close this PR and create a fresh with with RN content only.

**Comment by tmulquee** [comment]
PS. The unwanted content rightly lives in TELCODOCS-365 - it slipped into
the RN task due to an error of mine.
/tonym

**Comment by tmulquee** [comment]
The merge of the text developed in this PR was merged in a separate PR by Steven Smith. This PR can therefore be closed.
